.pre-commit-config.yaml

```diff
@@ -0,0 +1,15 @@
+---
+repos:
+  - repo: https://github.com/pre-commit/pre-commit-hooks
+    rev: v4.6.0
+    hooks:
+      - id: check-merge-conflict
+      - id: check-yaml
+      - id: check-xml
+      - id: end-of-file-fixer
+      - id: mixed-line-ending
+      - id: trailing-whitespace
+  - repo: https://github.com/jorisroovers/gitlint
```

Comment:
I understand the idea of some semantic convention in the git commit history to help with creating releases, though I normally tend to forget about those enforcements, which causes me more friction when I try to `amend` changes.

We can give a swing

Reply:
IMO, we can remove it again. I just took inspiration from oblt-infra.

 If we are going to follow our SemVer + automated versioning google doc, then we probably don't need this.

Reply:
WDYT?

Reply:
https://github.com/elastic/oblt-actions/pull/1/commits/68d6e7c090723ea3578da8a53ed13550f32ecab9